modules/gapi/perf/common/gapi_core_perf_tests_inl.hpp

```diff
@@ -1875,8 +2096,11 @@ PERF_TEST_P_(LUTPerfTest, TestPerformance)
     }
 
     // Comparison ////////////////////////////////////////////////////////////
-    EXPECT_EQ(0, cv::norm(out_mat_ocv, out_mat_gapi, NORM_INF));
-    EXPECT_EQ(out_mat_gapi.size(), sz_in);
+    {
+        EXPECT_TRUE(cmpF(out_mat_ocv, out_mat_gapi));
+        EXPECT_EQ(0, cv::norm(out_mat_ocv, out_mat_gapi, NORM_INF));
+        EXPECT_EQ(out_mat_gapi.size(), sz_in);
```

Comment:
Please leave only one `EXPECT_TRUE(cmpF(out_mat_gapi, out_mat_ocv));` and remove the rest.

Reply:
Applied, thanks